content/en/docs/tasks/administer-cluster/memory-manager.md

```diff
@@ -0,0 +1,390 @@
+
+Also, avoid the following configurations:
+1. duplicates, i.e. the same NUMA node or memory type, but with a different value;
+2. setting zero limit for any of memory types;
+3. NUMA node IDs that do not exist in the machine hardware;
+4. memory type names different than `memory` or `hugepages-<size>` (hugepages of particular `<size>` should also exist).
+
+Syntax: 
+
+`--reserved-memory N:memory-type1=value1,memory-type2=value2,...`
+* `N` (integer) - NUMA node index, e.g. `0`
+* `memory-type` (string) - represents memory type:
+  * `memory` - conventional memory
+  * `hugepages-2Mi` or `hugepages-1Gi` - hugepages 
+* `value` (string) - the amount of reserved memory, e.g. `1Gi`
+
+Example usage:
+
+`--reserved-memory 0:memory=1Gi,hugepages-1Gi=2Gi`
+
+or
+
+`--reserved-memory 0:memory=1Gi --reserved-memory 1:memory=2Gi`
+
+When you specify values for `--reserved-memory` flag, you must comply with the setting that you prior provided via Node Allocatable Feature flags. That is, the following rule must be obeyed for each memory type: 
+
+`sum(reserved-memory(i)) = kube-reserved + system-reserved + eviction-threshold`, 
+
+where `i` is an index of a NUMA node. 
+
+If you do not follow the formula above, the Memory Manager will show an error on startup.
+
+In other words, the example above illustrates that for the conventional memory (`type=memory`), we reserve `3Gi` in total, i.e.: 
+
+`sum(reserved-memory(i)) = reserved-memory(0) + reserved-memory(1) = 1Gi + 2Gi = 3Gi`
+
+An example of Node Allocatable Feature flags [configuration](https://kubernetes.io/docs/tasks/administer-cluster/out-of-resource/):
+* `--kube-reserved=cpu=500m,memory=50Mi`
+* `--system-reserved=cpu=123m,memory=333Mi`
+* `--eviction-hard=memory.available<500Mi`
+
+_NOTICE: hard eviction threshold is not equal to zero by default but `100Mi`, so do not forget to decrease the total amount set via `--reserved-memory` by this `100Mi`. Otherwise, the Memory Manager will display an error. Here is an example of a correct configuration:_
+
+```
+--feature-gates=MemoryManager=true 
+--kube-reserved=cpu=4,memory=4Gi 
+--system-reserved=cpu=1,memory=1Gi 
+--memory-manager-policy=static 
+--reserved-memory 0:memory=3Gi --reserved-memory 1:memory=2148Mi
+```
+Let us validate the configuration above:
+1. `kube-reserved + system-reserved + eviction-hard(default) = reserved-memory(0) + reserved-memory(1)`
+2. `4Gi + 1Gi + 100Mi = 3Gi + 2148Mi`
+3. `5120Mi + 100Mi = 3072Mi + 2148Mi`
+4. `5220Mi = 5220Mi` (correct!)

```

Comment:
In kubernetes configuration these values are dimensionless, but when checking our working we can use actual units:
```suggestion
2. `4GiB + 1GiB + 100Mi = 3GiB + 2148MiB`
3. `5120MiB + 100MiB = 3072MiB + 2148MiB`
4. `5220MiB = 5220MiB` (which is correct)
```

Reply:
What if I do not add "B" suffix here? Can somebody understand it differently than expected?
I understand that this is only a convention used in the official docs, right? E.g. here: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/ 
in description:
> For example, on a system where the default page size is 4KiB, you could specify a limit, hugepages-2Mi: 80Mi. If the container tries allocating over 40 2MiB huge pages (a total of 80 MiB), that allocation fails.

Did you skip "B" in 100Mi on purpose? Okay, it's typo...

Reply:
“100Mi” is a Kubernetes [_quantity_](https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/); “100MiB” is an amount of memory.

We can actually talk about that more explicitly.